doc/system-telemetry/reboot-cause.md

```diff
@@ -63,54 +69,85 @@ comment                 = STRING                         ; unstructured json for
 
 Currently `show reboot-cause` displays the last reboot-cause and performing `cat /host/reboot-cause/previous-reboot-cause.txt` to show the reboot-cause.
 This will be same as current design.
```

Comment:
As above, this seems redundant. `show reboot-cause` can now simply display the most recent reboot cause in State DB. Is there any good reason for retaining this?

Reply:
@jleveque The original reason that we planed to have two services which determines the reboot cause and process the reboot cause is to determine the reboot cause as early as possible but the process of the root-cause (saving to db) could be delayed after db  is ready. If we just want to refer to the db for the cli, there is no reason to maintain two service, I think. What do you think?